src/ResourceManager/Resources/Commands.Resources/ActiveDirectory/Models/ActiveDirectoryClient.cs

```diff
@@ -0,0 +1,818 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.Commands.ResourceManager.Common.Paging;
+using Microsoft.Azure.Graph.RBAC;
+using Microsoft.Azure.Graph.RBAC.Models;
+using Microsoft.Rest.Azure;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net;
+using ProjectResources = Microsoft.Azure.Commands.Resources.Properties.Resources;
+
+namespace Microsoft.Azure.Commands.ActiveDirectory
+{
+    public class ActiveDirectoryClient
+    {
+        public GraphRbacManagementClient GraphClient { get; private set; }
+
+        /// <summary>
+        /// Creates new ActiveDirectoryClient using WindowsAzureSubscription.
+        /// </summary>
+        /// <param name="context"></param>
+        public ActiveDirectoryClient(IAzureContext context)
+        {
+            GraphClient = AzureSession.Instance.ClientFactory.CreateArmClient<GraphRbacManagementClient>(
+                context, AzureEnvironment.Endpoint.Graph);
+
+            GraphClient.TenantID = context.Tenant.Id.ToString();
+        }
+
+        public PSADObject GetADObject(ADObjectFilterOptions options)
+        {
+            PSADObject result = null;
+
```

Comment:
remove extra vertical space - only after ending brackets '}' is it appropriate inside a method